winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -136,12 +149,11 @@ private static String getEndpointUrl(String address, Boolean useHttps, Integer p
         }
     }
 
-    @Deprecated
-    public static WinRmTool connect(String address, String username, String password) {
```

Comment:
Should put in the next release notes that we've deleted this? (but looks like it's been deprecated since Feb 2016 so that's good).